blockchain/fullblocktests/generate.go

```diff
@@ -575,7 +579,22 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	//   incorrectly ordered txouts, scripts that do not involve p2pkh or
 	//   p2sh addresses, etc
 	// - Try to vote with an immature ticket
-	// - Try to vote before stake validation height
+
+	// Create block that tries to vote before stake validation height.
+	//   bse#
+	//       \ -> bis1
+	g.NextBlock("bis1", nil, nil, func(b *wire.MsgBlock) {
+		// block bse0 being the first block to purchase tickets will be the
+		// first block with mature tickets.
+		voteBlock := g.BlockByName("bse0")
```

Comment:
This should be voting on the previous block `bseTipName`, not bse0 which would not be a valid vote even if the chain were after SVH at this point.  Recall that the intent of these tests is to ensure everything else about the block is valid except for the very specific failure condition being tested.